packages/core/src/interfaces/axis-scales.ts

```diff
@@ -109,6 +109,10 @@ export interface BasedAxisOptions {
 	 * axis that's being broken into bins
 	 */
 	binned?: boolean
+	/**
```

Comment:
The `correspondingDatasets` property belongs in AxisOptions rather than ComboChartAxisOptions because it's available for line and scatter charts as well. For a long time, there have been demo chart configs that use this property (and displayed an error in the IDE because the property didn't exist).